packages/shared/src/components/sidebar/SidebarDesktop.tsx

```diff
@@ -13,7 +13,7 @@ import { NetworkSection } from './sections/NetworkSection';
 import { CustomFeedSection } from './sections/CustomFeedSection';
 import { DiscoverSection } from './sections/DiscoverSection';
 import { ResourceSection } from './sections/ResourceSection';
-import { BookmarkSection } from './sections/BookmarkSection';
+import { BookmarkFolderSection } from './sections/BookmarkFolderSection';
```

Comment:
Not sure why the rename was needed, but not blocking

Reply:
Renamed back to `BookmarkSection`, this wasn't a key point for this PR ✔️